packages/manager/apps/dedicated/client/app/ip/ip/ip-ip.html

```diff
@@ -11,16 +18,37 @@ <h4 data-translate="ip_addresses_grid_header"></h4>
         data-track-click="trackClick"
         data-track-page="trackPage"
         data-tracking-data="trackingData"
+        data-params="{ serviceName: selectedService ? selectedService.serviceName : null }"
     >
-        <ip-list-header data-ng-if="serviceTypes.length">
-            <select
-                class="form-control"
-                id="serviceTypeSelector"
-                name="serviceTypeSelector"
-                data-ng-model="selectedServiceType"
-                data-ng-change="selectServiceType(selectedServiceType)"
-                data-ng-options="serviceType as (('ip_filter_service_type_' + serviceType) | translate) for serviceType in serviceTypes"
-            ></select>
+        <ip-list-header>
+            <div class="d-flex flex-column flex-lg-row">
+                <select
+                    class="form-control mr-3 mb-3"
+                    id="serviceTypeSelector"
+                    name="serviceTypeSelector"
+                    data-ng-model="ipScope.selectedServiceType"
+                    data-ng-change="ipScope.select({ serviceType: ipScope.selectedServiceType, service: null })"
+                    data-ng-options="serviceType as (('ip_filter_service_type_' + serviceType) | translate) for serviceType in ipScope.serviceTypes track by serviceType"
+                >
+                    <option
+                        value=""
+                        data-ng-bind="'ip_filter_service_type_all' | translate"
+                    ></option>
+                </select>
+                <select
+                    class="form-control"
+                    id="serviceSelector"
+                    name="serviceSelector"
+                    data-ng-model="ipScope.selectedService"
+                    data-ng-change="ipScope.select({ service: ipScope.selectedService, serviceType: null })"
+                    data-ng-options="service as service.serviceName group by (('ip_filter_services_title_' + service.category) | translate) for service in ipScope.services track by service.serviceName"
+                >
+                    <option
+                        value=""
+                        data-ng-bind="'ip_filter_services__ALL' | translate"
```

Comment:
```suggestion
                        data-ng-bind="'ip_filter_services_all' | translate"
```

Reply:
Hello @anooparveti 
I've reused an old key there
I you take a look at the translation file https://github.com/ovh/manager/blob/feat/ip-strat-add-service-list/packages/manager/apps/dedicated/client/app/ip/translations/Messages_en_GB.json#L78-L81 this is what we have
```json
  "ip_filter_services__ALL": "All services",
  "ip_filter_services__ALERTS": "Services under alert",
  "ip_filter_services__PARK": "IP parking",
  "ip_filter_services__FAILOVER": "Additional IP",
```